docs/fsharp/tutorials/getting-started/getting-started-command-line.md

```diff
@@ -14,13 +14,13 @@ ms.assetid: 615db1ec-6ef3-4de2-bae6-4586affa9771
 
 # Getting started with F# with command-line tools
 
-This article covers how you can get started with using F# on .NET Core with the ..NET Core SDK 1.0 Preview 2.  It will go through building a multi-project solution with a Class Library, a Console App, and an xUnit test project.
+This article covers how you can get started with using F# on .NET Core 1.0 with the .NET Core SDK 2.0.1 Preview 2.  It will go through building a multi-project solution with a Class Library that is called by a Console Application.
```

Comment:
I thought the latest versions are [.Net Core 1.0.1 and (deep breath) ".NET Core 1.0.0 SDK - Preview 2 (build 003131)"](https://github.com/dotnet/core/releases/tag/1.0.1). I certainly didn't see it being called 2.0.1 anywhere.